__tests__/spec/sanity-checks.js

```diff
@@ -22,6 +22,7 @@ function readFile (pathFromRoot) {
  */
 describe('The Prototype Kit', () => {
   beforeAll(() => {
+    jest.spyOn(sass, 'compile').mockImplementation((css, options) => ({ css }))
```

Comment:
This makes me nervous; why do the tests pass when we're stubbing out a core function like this?

Reply:
Could you add more detail to the commit message on why this change is required?